### DISCUSSION TITLE
Achievements information should mention Hardcore/softcore.

### DISCUSSION THREAD

**Discussion by Kinglink** [RetroAchievements/RAWeb#1860] [Ideas]
![image](https://github.com/RetroAchievements/RAWeb/assets/2449488/f007b63b-621d-43ae-9557-2c309656c6c6)

It looks like there's no verbal indication of whether this is a hard core or softcore set. 

I believe the banner around the game should indicate it's status, but on a known earned softcore achievement it looks the same. 

![image](https://github.com/RetroAchievements/RAWeb/assets/2449488/4595a87c-ba84-4350-9c2a-1a8044bfd174)

Maybe changing the Unlock message to "Unlocked in Softcore 07 Feb, 2023 09:30" for the SC achievements would differentiate it?

**Comment by Alexdatadestroyer** [comment]
Agreed. This is a minor information to add. I don't think it would take long to be implemented on future versions of RAWeb...

However, at achievement page footer, you can see if the user got an achievement on Hardcore Mode or not - which is a thing that differentiate unlock status already.

**Comment by Sanaki** [comment]
We've never actually used the term softcore directly on the site or in emulators. Recently we've been talking about rebranding it as something like "relaxed mode". If we go so far as specifically saying the mode, we'll want to finish deciding on the term first.

**Reply by Kinglink** [reply]
I don't wish to disagree with you @Sanaki, but I believe you're incorrect on this, at least after the redesign, and it's active today. 
Upper right corner, mouse over the grey. 
![image](https://github.com/RetroAchievements/RAWeb/assets/2449488/6b6478a1-037e-490e-9e84-072f3784e6af)

In addition if you earned the achievement in softcore mode it's listed on the new Progress box.; 

![image](https://github.com/RetroAchievements/RAWeb/assets/2449488/43812a68-5adf-41ae-909b-e0a611ac85e9)

I have no problem if we use a different terminology or say "Unlocked IN HARDCOREEEEEEEE"  instead of calling out the Softcore unlock.   But the point is more they should be different. 

Also while looking at this I noticed on the Game Info page, all achievements are only listed as "Unlocked"  
![image](https://github.com/RetroAchievements/RAWeb/assets/2449488/3fdc0fc9-8e1d-4c28-b027-0c87f1d716b2)

PS. Relaxed/Casual/Fun mode, all do sound a lot better than "Softcore."

**Reply by Sanaki** [reply]
Yeaaah... I already had someone point out that I was wrong. :P Regardless, we do need to rebrand it soon. Especially since it's already been added in a couple places.

**Comment by Jamiras** [comment]
The achievement image should be adorned with a gold border to indicate hardcore. If you're going to use web extensions to modify the appearance of the webpage, expect to run into some oddities like this.

Maybe that's changed since this was originally opened?

**Reply by tylerh111** [reply]
The gold boarder is there, but it can be hard to recognize that it's unlocked in hardcore mode if the badge itself has a gold board, e.g. all the zelda alttp achievement badges.

Would it be possible to add another indicator (like mentioned above with adding which mode it was unlocked in) for hardcore unlocked achievements? Maybe adding a space between the badge and achievement information that shows an icon depending on the state of the achievement: a grey lock for locked, a green open lock for softcore unlocked, and a gold star for hardcore unlocked.

**Comment by Chew** [comment]
I believe this is a thing now with the new redesigned page. Examples with my fake seed data. Now clearly shows unlocked or unlocked with hardcore. And even shows how many unlocked in each mode.

<img width="1746" height="484" alt="CleanShot 2026-05-22 at 20 03 32@2x" src="https://github.com/user-attachments/assets/629f2f24-470d-4b77-b83d-92931a2b18a5" />
<img width="1768" height="502" alt="CleanShot 2026-05-22 at 20 02 58@2x" src="https://github.com/user-attachments/assets/943b67b2-45ae-4916-81d6-9667516ea9e0" />

**Comment by tylerh111** [comment]
Yes, the info on the achievement page is now explicit. I was thining it could be made clearer on the game page.

I don't have any softcore achievement only on my account, a good example is the zelda alttp achievement icons which have a border. So it can be tough to check which hardcore achievements are missing.

<img width="881" height="493" alt="Screenshot From 2026-06-13 01-25-35" src="https://github.com/user-attachments/assets/04839108-ded6-4140-b4af-6d5a694f886c" />

Maybe the following could help, where the achievement icons are slightly smaller (or the border is bigger) so the border is clearer. Or have a dot indicating lock status (locked, unlocked, hardcore unlocked).

<img width="353" height="184" alt="Screenshot From 2026-06-13 01-25-35 (2)" src="https://github.com/user-attachments/assets/761a90c9-84ac-405b-ade9-23eb62bcf5ff" />

**Reply by wescopeland** [reply]
We already have a "Locked Only" filter. It disappears if you've mastered the game because at that point it's redundant.

**Reply by tylerh111** [reply]
Cool, I didn't realize locked did that. All good for me then. Thanks!
